### PR TITLE
Typography changes

### PR DESCRIPTION
**Pull request by maxilev**

### CODE REVIEW

File: sass/core.scss [hunk 1]
```diff
@@ -408,7 +429,7 @@
   .yotpo-icon-btn-small {
     display: inline-block;
     .yotpo-icon {
-      
+
```
Comment (zanamich) [inline sass/core.scss]: @maxilev  ?

Comment (maxilev) [inline sass/core.scss]: ?

**Comment by zanamich** [comment]
@gadinehab  very good

**Comment by dmitri86git** [comment]
@NetanelMoshkovitz :+1: 
Have connected branches:
https://github.com/YotpoLtd/yotpo-api/pull/829
https://github.com/YotpoLtd/yotpo-b2b/pull/538
https://github.com/YotpoLtd/yotpo-widget/pull/552

**Pull request merged**